src/main/java/com/epam/test_generator/dto/CaseDTO.java

```diff
@@ -34,6 +34,11 @@
     @Valid
     private Set<TagDTO> tags;
 
+    @NotNull
+    @Min(value = 0)
+    @Max(value = 4)
```

Comment:
Hibernate allows to store enums as varchars, doing this will decrease usage of magical numbers.